modular_skyrat/master_files/code/modules/admin/admin.dm

```diff
@@ -45,3 +45,24 @@ GLOBAL_VAR_INIT(dchat_allowed, TRUE)
 
 	handler = "[usr.ckey]"
 	return TRUE
+
+///Procedure whick convert adminhelp to mentorhelp
+/datum/admin_help/proc/convert(key_name = key_name_admin(usr))
+	if(state != AHELP_ACTIVE)
+		return FALSE
+	
+	if(handler && handler != usr.ckey)
+		var/response = tgui_alert(usr, "This ticket is already being handled by [handler]. Do you want to continue?", "Ticket already assigned", list("Yes", "No"))
+		if(!response || response == "No")
+			return FALSE
+	
+	add_verb(initiator, /client/verb/mentorhelp)
```

Comment:
Why are you giving them the verb? I'm pretty sure everyone gets that verb by default, no?